.github/workflows/pr.yml

```diff
@@ -183,7 +183,7 @@ jobs:
         run: nuget restore ${{ runner.temp }}\testcli\windows\testcli.sln
 
       - name: Build testcli.sln
-        run: msbuild ${{ runner.temp }}\testcli\windows\testcli.sln /nologo /nr:false /p:PreferredToolArchitecture=x86 /p:platform="%BUILDPLATFORM%" /p:configuration="%BUILDCONFIGURATION%" /clp:NoSummary;NoItemAndPropertyList;Verbosity=normal
```

Comment:
>PreferredToolArchitecture=x86  [](start = 89, length = 30)

For my education: Why did we prefer x86 here while elsewhere we seem to have preferred x64?

Reply:
Very likely an oversight. We don't actually run Github Actions based CI right now.